legacy/full/Dockerfile

```diff
@@ -183,26 +183,26 @@ ENV PATH=/workspace/.rvm/bin:$PATH
 LABEL dazzle/layer=lang-rust
```

Comment:
Please revert these changes. We are not updating the legacy images anymore.

Reply:
Oh yeah, missed this part. Thank you for spotting 🙏